src/client/components/Avatar/Avatar.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Avatar = (props) => {
+  return <div className="all-avatar-images">{props.image}</div>;
```

Comment:
This can be done with src and alt as props
for example 
```
export const Avatar = (props) => {
  return (
    <img className="avatar"
      src={props.avatarUrl}
      alt={props.name}
    />
  );
}
```
An image can also be wrapped with a div if that's the requirement.